MantidQt/MantidWidgets/src/MantidWSIndexDialog.cpp

```diff
@@ -108,15 +107,15 @@ QMultiMap<QString, std::set<int>> MantidWSIndexWidget::getPlots() const {
  * @returns True if waterfall plot selected
  */
 bool MantidWSIndexWidget::isWaterfallPlotSelected() const {
-  return m_waterfallOpt ? m_waterfallOpt->isChecked() : false;
+  return (m_plotOptions->currentText() == "Waterfall Plot");
```

Comment:
I don't know if this will ever be an issue for us, but the 'Waterfall Plot' string is run through the `tr()` function when items are added to the combo box. Is there a possibility for `m_plotOptions->currentText()` to return something else than 'Waterfall Plot' or 'Tiled Plot' if Mantid is ever translated to another language?